### PR TITLE
fix(KFLUXBUGS-1328): pull secrets in rpm-ostree

### PR DESCRIPTION
**Pull request by scoheb**
- copy functionality used in buildah-remote task

### CODE REVIEW

**Review by chmeliik — APPROVED** [review]
LGTM, thanks :+1:

File: task/rpm-ostree/0.1/rpm-ostree.yaml [hunk 1]
```diff
@@ -173,6 +174,7 @@ spec:
           -e COMMIT_SHA="$COMMIT_SHA" \
           --rm \
           -v "$BUILD_DIR/workspaces/source:$(workspaces.source.path):Z" \
+          -v "$BUILD_DIR/.docker/:/root/.docker:Z" \
```
Comment (cgwalters) [inline task/rpm-ostree/0.1/rpm-ostree.yaml:177]: I'm a bit confused by this one, is it really necessary?  It seems to expose the pull secret to the container itself, but I don't think we need that here FWIW.

Comment (chmeliik) [reply task/rpm-ostree/0.1/rpm-ostree.yaml:177]: Oh good catch, this is a different use case than the buildah-remote task. This one wants to use the credentials to pull the BUILDER_IMAGE itself, and there's no need to pass the credentials to that image again

I think we need to rsync the docker credentials to the $HOME of the VM user

**Comment by cgwalters** [comment]
There's a more medium term thing here that I think we can actually kill the rpm-ostree task in favor of buildah-remote.

File: task/rpm-ostree/0.1/rpm-ostree.yaml (hunk 1 above)
Comment (stuartwdouglas) [reply task/rpm-ostree/0.1/rpm-ostree.yaml:177]: You need to either sync the credentials to the $HOME/.docker dir, or explicitly tell podman which credentials file to use.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

**Comment by lmilbaum** [comment]
This PR is a blocker for a downstream MR. Would you be able to prioritize this?

**Comment by chmeliik** [comment]
looks abandoned, can we close this?

**Comment by scoheb** [comment]
Yep. We can close.

**Pull request closed without merging**